### ISSUE TITLE
chore: obtain OpenSSF Best Practices badge (CII-Best-Practices)

### ISSUE THREAD

**Issue by shinagawa-web** [shinagawa-web/gomarklint#303]
## Overview

The OpenSSF Scorecard flags this repository for not having an [OpenSSF Best Practices badge](https://www.bestpractices.dev/). Earning the badge closes Scorecard alert #44 (CII-Best-Practices) and demonstrates the project's security and quality maturity.

## Steps

1. Go to https://www.bestpractices.dev/ and sign in with your GitHub account.
2. Click **Add project** and enter the repository URL: `https://github.com/shinagawa-web/gomarklint`
3. Answer the questionnaire (~100 questions covering documentation, testing, security practices, etc.).
4. Once the badge is issued, add it to `README.md` and `README.ja.md`:

```markdown
[![OpenSSF Best Practices](https://www.bestpractices.dev/projects/XXXX/badge)](https://www.bestpractices.dev/projects/XXXX)
```

(Replace `XXXX` with the project ID assigned after registration.)

5. The Scorecard workflow will automatically detect the badge URL in the README and close alert #44 on the next scan.

## References

- OpenSSF Best Practices program: https://www.bestpractices.dev/
- Scorecard check definition: https://github.com/ossf/scorecard/blob/main/docs/checks.md#cii-best-practices
